shell/platform/android/io/flutter/embedding/android/FlutterActivity.java

```diff
@@ -779,8 +780,11 @@ public String getDartEntrypointFunctionName() {
    * have control over the incoming {@code Intent}.
    *
    * <p>Subclasses may override this method to directly control the initial route.
+   *
+   * <p>If this method returns null and the {@code shouldHandleDeeplinking()} returns true, the
+   * {@link FlutterActivityAndFragmentDelegate} retrieves the initial route from the {@code Intent}
```

Comment:
don't link to package private classes from public methods' docs. The fact that FlutterActivity delegates to a class is an implementation detail and not relevant to the activity's users.

Reply:
just curious, is there a place like api.flutter.dev that shows these doc string?

Reply:
not sure if I understood the question correctly. Do you mean these? https://api.flutter.dev/javadoc/

Reply:
yes, thanks for the link